.fixtures.yml

```diff
@@ -10,6 +10,12 @@ fixtures:
       repo: git://github.com/puppetlabs/puppetlabs-yumrepo_core
       ref: 1.0.1
       puppet_version: ">= 6.0.0"
+    # Need by postgresql
+    concat:
+      repo: git://github.com/puppetlabs/puppetlabs-concat.git
```

Comment:
this should use a tag

Reply:
Added a tag here.

Reply:
tag is not there.. forgot to push?

Reply:
Fixed, not sure where it ended up but also squashed commits